groovy/postInit/mod/MachineRecipes.groovy

```diff
@@ -980,7 +980,8 @@ recipemap('assembler').recipeBuilder()
 //Power Units
 recipemap('assembler').recipeBuilder()
 		.inputs(ore('gearSmallSteel') * 2)
-		.inputs(ore('boltSteel') * 2)
+		.inputs(ore('screwSteel'))
+		.inputs(ore('plateSteel') * 2)
 		.inputs(metaitem('electric.motor.lv') * 2)
```

Comment:
this should only cast 1 motor
![image](https://github.com/user-attachments/assets/fc705867-5dee-4d01-9769-a60f396c6ebc)

Reply:
![image](https://github.com/user-attachments/assets/4ca08475-b0a0-4017-9d56-30891511ddee)